mainapp/tests/api/test_get_info.py

```diff
@@ -43,8 +43,7 @@ def test_get_info_success(self):
         self.assertEqual(data["lon"], self.model1.location.longitude)
         self.assertEqual(data["license"], self.model1.license)
         self.assertEqual(data["desc"], self.model1.description)
-        self.assertEqual(data["author"], self.model1.author.username)
-        # self.assertEqual(data['date'], self.model1.upload_date.isoformat()) # Check format if needed
+        self.assertEqual(data["author"], self.model1.author.profile.uid)
```

Comment:
Can you please in this test change to test against a literal instead of the attribute from `self.model1`?

The background here is that I see in the documentation above that "author" will still come back as a string, not as an integer. That's probably sensible, given that it was a string before. But I would want to have a test that fails when the output suddenly changes to an integer type.

And the same argument applies to the other fields in the response.

Reply:
That's okay.